packages/utils/src/internals/error_snapshotter.ts

```diff
@@ -0,0 +1,100 @@
+import crypto from 'node:crypto';
+
+import type { CrawlingContext } from '@crawlee/basic';
+import type { KeyValueStore } from '@crawlee/core';
+import type { Page as PlaywrightPage } from 'playwright';
+import type { Page as PuppeteerPage } from 'puppeteer';
+
+import type { ErrnoException } from './error_tracker';
+
+/**
+ * ErrorSnapshotter class is used to capture a screenshot of the page and a snapshot of the HTML when an error occur during web crawling.
+ */
+export class ErrorSnapshotter {
+    static readonly MAX_ERROR_CHARACTERS = 30;
+    static readonly BASE_MESSAGE = 'An error occurred';
+    static readonly SNAPSHOT_PREFIX = 'SNAPSHOT';
```

Comment:
I'm voting for `ERROR_SNAPSHOT`. It is a bit more verbose but at least no one will guess what these files are.